app/controllers/courses_controller.rb

```diff
@@ -18,7 +18,8 @@ def edit
   end
 
   def create
-    CourseCreationService.new(course_params, current_instructor.id, self).create_course
+    CourseCreationService.new(course_params.except(:tags), current_instructor.id, self,
```

Comment:
Pass only `course_params` and inside service use
course.update(course_params.except(:tags))